Quaver.Shared/Screens/Selection/UI/FilterPanel/Search/FilterPanelSearchBox.cs

```diff
@@ -121,5 +128,17 @@ private void StoppedTyping(string filter)
             CurrentSearchQuery.Value = filter;
             PreviousSearchTerm = filter;
         }
+
+        /// <summary>
+        ///		Called when the user types a character in the textbox.
+        ///	</summary>
+        private void CharacterTyped(char c)
```

Comment:
Rather than adding this functionality for one single textbox, it should be implemented for all of them at framework level. Ideally when the skin loads, the list of AudioSamples should be cached for *every* textbox to use. Then each individual textbox can decide if they want to play the sounds or not (on by default).

Reply:
That sounds good actually, I didn't think of the other textboxes.